src/main/resources/org/csanchez/jenkins/plugins/kubernetes/pipeline/PodTemplateStep/config.jelly

```diff
@@ -24,7 +24,7 @@
 
   <f:entry field="inheritFrom" title="${%Pod template to inherit from}"
     help="/plugin/kubernetes/help/inheritFrom.html">
-    <f:textbox/>
+    <f:textbox default=",default,"/>
```

Comment:
Best to share this constant with the Java code by referring to some (possibly static) field of the descriptor, e.g.

```suggestion
    <f:textbox default="${descriptor.defaultInheritFrom}"/>
```

[Guide](https://jenkins.io/doc/developer/plugin-development/pipeline-integration/#handling-default-values)

Reply:
(plus use of the same constant in `setInheritFrom` of course)